adminSiteClient/GdocsPreviewPage.tsx

```diff
@@ -233,9 +234,46 @@ export const GdocsPreviewPage = ({ match, history }: GdocsMatchProps) => {
                     <GdocsDiff originalGdoc={originalGdoc} gdoc={gdoc} />
                 </Drawer>
 
-                <DebugProvider>
-                    <OwidArticle {...gdoc} />
-                </DebugProvider>
+                {/* 
+                    This uses the full SSR rendering pipeline. It is more accurate but comes
+                    with an additional requests to the Google API and has a less polished
+                    authoring experience at the moment (content flashes and scrolling position
+                    resets on every change)
+                */}
+                <iframe
+                    src={`/gdocs/${gdoc.id}/preview#owid-article-root`}
+                    style={{ width: "100%", height: "inherit", border: "none" }}
+                    key={gdoc.revisionId}
+                />
+
+                {/* Operational alternative for preview rendering, with some unresolved issues.
```

Comment:
Soft preference to not bother keeping this part in. 🙂 

I think either a) we'll know in the next month or so if this solution ends up not working for us (in which case you'll be able to reference this branch still) or b) we'll be fine with it and all these comments will just hang around without a clear expiry date.